class.jetpack-modules-list-table.php

```diff
@@ -115,13 +115,15 @@ function get_views() {
 			'all' => sprintf( $format, $title, $count, $url, $current ),
 		);
 		foreach ( $module_tags_unique as $title => $count ) {
-			$key           = sanitize_title( $title );
-			$display_title = esc_html( wptexturize( $title ) );
-			$url           = add_query_arg( 'module_tag', urlencode( $title ) );
-			$current       = '';
-			if ( ! empty( $_GET['module_tag'] ) && $title == $_GET['module_tag'] )
-				$current   = ' class="current"';
-			$views[ $key ] = sprintf( $format, $display_title, $count, $url, $current );
+			if( 'Jumpstart' !== $title ) {
```

Comment:
I'm not crazy about using this logic to determine if you're in Jumpstart... this would break if we change it to, say, "Jump Start" or "JumpStart" or "Quick Start" or whatever.

I'm not super familiar with this, though, can you think of another way to determine, @jessefriedman ?  Feel free to ping me on Slack and we can discuss